exercises/concept-wip/crystal-hunter/.meta/config.json

```
@@ -0,0 +1,12 @@
+{
+  "authors": ["meatball133"],
+  "contributors": [],
+  "files": {
+    "solution": ["src/crystal_hunter.cr"],
+    "test": ["spec/crystal_hunter_spec.cr"],
+    "example": [".meta/src/exemplar.cr"]
+  },
+  "forked_from": ["elixir/pacman-rules"],
+  "icon": "low-power-embedded-game",
+  "blurb": "Learn about booleans by implementing the rules of the game, Crystal Hunter."
```

```suggestion
  "blurb": "Learn about booleans by implementing the rules of the Crystal Hunter game."
```

I think the comma is not needed here.